pkgs/development/tools/cpm/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "cpm";
+  version = "0.35.1";
+
+  src = fetchFromGitHub {
+    owner = "cpm-cmake";
+    repo = "CPM.cmake";
+    rev = "v${version}";
+    hash = "sha256-Oon/5iwkUUASsUDvde69iEwLe8/CAzwYKYsyzH5K+V0=";
+  };
+
+  dontConfigure = true;
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p ${placeholder "out"}/share/cpm/
+    cp ./cmake/CPM.cmake ${placeholder "out"}/share/cpm/
```

Comment:
Remember to decorate custom phases, surrounding the code with `runHook pre<phase name>` and  `runHook post<phase name>`.

Reply:
Ah, thank you for pointing it out!